sphinx/ext/coverage.py

```diff
@@ -65,6 +66,44 @@ def _add_row(col_widths: list[int], columns: list[str], separator: str) -> Itera
     yield _add_line(col_widths, separator)
 
 
+def load_modules(mod_name: str, ignored_module_exps: list[re.Pattern[str]]) -> set[str]:
```

Comment:
Actually, do you want the function to ne part of tue public API? or could we rename it `_load_modules`?

Reply:
Fair point. Done.